worldedit-core/src/main/java/com/sk89q/worldedit/extension/factory/parser/DefaultBlockParser.java

```diff
@@ -206,7 +207,36 @@ private static Map<Property<?>, Object> parseProperties(BlockType type, String[]
     public Stream<String> getSuggestions(String input, ParserContext context) {
         final int idx = input.lastIndexOf('[');
         if (idx < 0) {
-            return SuggestionHelper.getNamespacedRegistrySuggestions(BlockType.REGISTRY, input);
+            ArrayList<String> playerSpecificBlocks = new ArrayList<>();
```

Comment:
This should be `List<String> playerSpecificBlocks = new ArrayList<>();` to conform with general styling